ProjectSettings/ProjectSettings.asset

```diff
@@ -635,7 +638,7 @@ PlayerSettings:
   metroCertificatePassword: 
   metroCertificateSubject: Microsoft
   metroCertificateIssuer: Microsoft
-  metroCertificateNotAfter: 0090761b2802d501
+  metroCertificateNotAfter: 00eb1f507373d501
```

Comment:
Probably shouldn't be changing the default project settings

Reply:
Do you know what that is? I'll see if I can change it back

Reply:
I wish there was a button here that just said "don't keep this change" and it would change it back.

Reply:
fixed this